src/lib.rs

```diff
@@ -213,6 +213,13 @@ impl Event {
     /// This is a wrapper of
     /// [tep_record_print_fields](https://www.trace-cmd.org/Documentation/libtraceevent/libtraceevent-field_print.html).
     pub fn print_fields(&self, rec: &Record) {
+        println!("fields: {:?}", self.get_fields(rec));
+    }
```

Comment:
nit: Can you add a blank line between the end of the previous method and the comment of the next method for consistency?